src/qililab/drivers/instruments/qblox/pulsar.py

```diff
@@ -25,15 +25,13 @@
 
 @InstrumentDriverFactory.register
 class Pulsar(QcodesPulsar, BaseInstrument):  # pylint: disable=abstract-method
-    """Qililab's driver for QBlox-instruments Pulsar"""
+    """Qililab's driver for QBlox-instruments Pulsar
+    Args:
```

Comment:
There should always be a blank line between the description and the arguments.

```suggestion
    """Qililab's driver for QBlox-instruments Pulsar

    Args:
```

Reply:
Thanks commited!

when tests pass I'll merge it!